inbc-program/README.md

```diff
@@ -114,9 +119,19 @@ inbc sota
      --releasedate 2022-02-22 
      --username <username>
 ```
-#### Edge Device on Ubuntu
+#### Edge Device on Ubuntu in update mode
 ```
-inbc sota
+inbc sota --command update
+```
+
+#### Edge Device on Ubuntu in download_only mode
+```
+inbc sota --command download-only
```

Comment:
I believe she did it this way, because in the manifest it has <cmd as the tag and we need it to be backwards compatible.  The word 'mode' makes more sense in this case, but I can see using command since that is what the manifest states already.

Reply:
But now that I see below that we are using cmd for the rollback, I agree that we should be using mode everywhere even if this get in the manifest under the cmd tag.

Reply:
@nmgaston I was thinking `inbc sota --command update --mode download-only` and that `--command update` was implied (default).

Reply:
(as the previous invocation was `inbc sota` by itself)

Reply:
@gblewis1  as mentioned in the story and also the email change I'll stick to inbc sota --mode download-only